README.md

```diff
@@ -1,15 +1,21 @@
 # Create Block Theme
 
-Welcome to Create Block Theme - a WordPress plugin to create block themes from within the Site Editor.
+Welcome to Create Block Theme - a WordPress plugin to create block themes from within the Editor.
 
-It works alongside features that are already available in the Site Editor to enhance the workflow for creating block themes. After being tested in this plugin, some of the features included here may be moved into the Site Editor itself.
+It works alongside features that are already available in the Editor to enhance the workflow for creating block themes. After being tested in this plugin, some of the features included here may be moved into the Editor itself.
 
 *Disclaimer:* Create Block Theme enables development features and thus is a tool that should be treated like as such; you can think of it as a Development Mode for WordPress, and you should keep in mind that changes made through this plugin could change your site and/or theme permanently.
 
 This plugin allows you to:
 
-- Export your existing theme with all customizations made in the Site Editor
-- Create a new theme, blank theme, child theme, or style variation from the Site Editor
+- Export your existing theme with all customizations made in the Editor
+- Create a new theme, blank theme, child theme, or style variation from the Editor
+
+This plugin also makes several changes to the contents of an exported theme, including:
+
+- Adds all images used in templates or pages to the theme's `assets` folder.
```

Comment:
pages? Did you mean patterns?  I think we should just mention templates as there really isn't a way to manage patterns (directly) currently.

Reply:
Huh, no I think I did mean pages but what I actually meant was templates. Will remove!